README.md

```diff
@@ -156,7 +155,7 @@ security:
             form_login:
                 login_path: contao_frontend_login
                 check_path: contao_frontend_login
-                default_target_path: contao_index
+                default_target_path: /
```

Comment:
What's the point of changing this and returning a redirect route in the controller instead?

Reply:
The `default_target_path` is used if no other redirect target can be determined (see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/Http/Authentication/DefaultAuthenticationSuccessHandler.php#L125). This is the case if the `mod_login` template is not adjusted and the `_target_path` hidden field is not submitted. Regardless of that, the target **path** can never be a route, so the configuration was incorrect in the first place.